### DISCUSSION TITLE
v0.2.0-alpha

### DISCUSSION THREAD

**Discussion by PussInScreen** [andrewk10/AutoCompliance#36] [Announcements]
## What's Changed
* Wiki and Documentation Establishment by @andrewk10 in https://github.com/andrewk10/AutoCompliance/pull/12
* sprint3 into develop for established documentation by @andrewk10 in https://github.com/andrewk10/AutoCompliance/pull/18
* develop to main for documentation purposes by @andrewk10 in https://github.com/andrewk10/AutoCompliance/pull/19
* [FIX] for help menue and added requirements.txt    by @0x30c4 in https://github.com/andrewk10/AutoCompliance/pull/21
* Adding all strings (where possible) to the strings.py script. by @andrewk10 in https://github.com/andrewk10/AutoCompliance/pull/20
* Tests for Net Propagation Functions by @andrewk10 in https://github.com/andrewk10/AutoCompliance/pull/25
* Sprint3 - v0.2.0 Release by @andrewk10 in https://github.com/andrewk10/AutoCompliance/pull/34
* Sprint3 - v0.2.0 Release by @andrewk10 in https://github.com/andrewk10/AutoCompliance/pull/35

## New Contributors
* @0x30c4 made their first contribution in https://github.com/andrewk10/AutoCompliance/pull/21

**Full Changelog**: https://github.com/andrewk10/AutoCompliance/compare/v0.1.0-alpha...v0.2.0-alpha

<hr /><em>This discussion was created from the release <a href='https://github.com/andrewk10/AutoCompliance/releases/tag/v0.2.0-alpha'>v0.2.0-alpha</a>.</em>

**Comment by 0x30c4** [comment]
Created a new branch for the argparser implementation.
The basic ground work is done but the function implementation is still left.
https://github.com/andrewk10/AutoCompliance/tree/update/adding-argparser

**Reply by PussInScreen** [reply]
> Created a new branch for the argparser implementation. The basic ground work is done but the function implementation is still left. https://github.com/andrewk10/AutoCompliance/tree/update/adding-argparser

No worries, I created a pull request to merge your work in with my work (when you're done of course), want to include all revised main.py/demo.py work as part of the overall refactor work being done. When you are done let me know and I'll review. :)
